hack/latest-kind-images.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The cert-manager Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -eu -o pipefail
+
+# source this lib for KIND_IMAGE_REPO
+source "devel/lib/lib.sh"
+
+CRANE=crane
+TAGS=$(mktemp)
+
+trap 'rm -f -- "$TAGS"' EXIT
+
+if ! which $CRANE >/dev/null 2>&1; then
+	echo -e "Couldn't find crane. Try running:\ngo install github.com/google/go-containerregistry/cmd/crane@latest" >&2
+	exit 1
+fi
+
+function latest_kind_tag () {
+	grep -E "^v$1" $TAGS | sort | tail -1
+}
+
+$CRANE ls $KIND_IMAGE_REPO > $TAGS
```

Comment:
Question: sometimes we will want to consume kind images from our own repo as we'll be wanting to test against a version of Kubernetes that doesn't have a published kind image yet (such as now with kube 1.23, see https://github.com/jetstack/testing/pull/604). What would be the best solution for that as I guess this script would overwrite a manually added sha/kind repo?

Reply:
I think it would be easy enough too modify this script to add that; failing any automation, I think we could still hardcode stuff in create-kind.sh easily!

Reply:
Awesome, I'll try that once this PR gets merged as I'll be adding 1.23